alf/utils/common.py

```diff
@@ -1481,7 +1478,14 @@ def get_alf_snapshot_env_vars(root_dir):
     points to the ALF snapshot under this directory.
     """
     unzip_alf_snapshot(root_dir)
-    alf_repo = os.path.join(root_dir, "alf")
+    legacy_alf_repo = os.path.join(root_dir, "alf")
```

Comment:
Just want to confirm my understanding: legacy mode and non-legacy mode go thought the same procedure in generation of snapshot, and the difference (one extra layer of directory) is handled when getting the snapshot. Is that correct?

Reply:
> Just want to confirm my understanding: legacy mode and non-legacy mode go thought the same procedure in generation of snapshot, and the difference (one extra layer of directory) is handled when getting the snapshot. Is that correct?

Not the same generation. The new mode copies only the 'alf' module, but not its parent dir.